context/src/main/java/io/micronaut/scheduling/TaskScheduler.java

```diff
@@ -32,32 +32,34 @@
      * Creates and executes a one-shot action that becomes enabled
      * after the given delay.
      *
-     * @param cron    The cron expression
-     * @param command the task to execute
+     * @param cron     The cron expression
+     * @param timezone The time zone to base the cron expression on. Defaults to system time zone
+     * @param command  the task to execute
      * @return a ScheduledFuture representing pending completion of
      * the task and whose {@code get()} method will return
      * {@code null} upon completion
      * @throws java.util.concurrent.RejectedExecutionException if the task cannot be
      *                                                         scheduled for execution
      * @throws NullPointerException                            if command or delay is null
      */
-    ScheduledFuture<?> schedule(String cron, Runnable command);
+    ScheduledFuture<?> schedule(String cron, String timezone, Runnable command);
```

Comment:
This is a public method and thus cannot be changed. A new default method will have to be created instead